docs/storage/azure-storage-queues-component.md

```diff
@@ -116,17 +116,18 @@ You can also pass the `Action<AzureStorageQueuesSettings> configureSettings` del
 ```csharp
 builder.AddAzureQueueService(
     "queue",
-    settings => settings.HealthChecks = false);
+    static settings => settings.HealthChecks = false);
 ```
 
 You can also set up the `QueueClientOptions` using `Action<IAzureClientBuilder<QueueServiceClient, QueueClientOptions>> configureClientBuilder` delegate, the second parameter of the <xref:Microsoft.Extensions.Hosting.AspireQueueStorageExtensions.AddAzureQueueService%2A> method. For example, to set the first part of user-agent headers for all requests issues by this client:
 
 ```csharp
 builder.AddAzureQueueService(
     "queue",
-    configureClientBuilder: clientBuilder =>
-        clientBuilder.ConfigureOptions(options =>
-            options.Diagnostics.ApplicationId = "myapp"));
+    static configureClientBuilder:
+        clientBuilder => clientBuilder.ConfigureOptions(
+            static options =>
+                options.Diagnostics.ApplicationId = "myapp"));
```

Comment:
I think you put this in the wrong spot, unless there's some new fancy shmancy C# feature I don't know about 😁 Of course, using snippets would have caught this error 😁 And don't get me started on your lack of VB samples!
```suggestion
    configureClientBuilder:
        static clientBuilder => clientBuilder.ConfigureOptions(
            static options =>
                options.Diagnostics.ApplicationId = "myapp"));
```